pkg/scheduler/framework/session_plugins.go

```diff
@@ -144,6 +144,7 @@ func (ssn *Session) Preemptable(preemptor *api.TaskInfo, preemptees []*api.TaskI
 	var init bool
 
 	for _, tier := range ssn.Tiers {
+		init = false
```

Comment:
why set init to false in every tier?

Reply:
The init might be set to true, however the intersection of tier1 might be nil. 

Take below as example, plugin1 get victims and set init true, plugin2 get nil. The intersection is nil. In this case, we want to run Tier2 plugin3 with new start(init is false)

Tier1 
plugin1
plugin2
Tier2
plugin3